main-site-utilities.php

```diff
@@ -20,3 +27,7 @@
 
 	WP_CLI::add_command( 'research', 'UCF\MainSiteUtilities\Commands\ResearchCommands' );
 }
+
+add_action( 'plugins_loaded', function() {
+	add_action( 'init', array( __NAMESPACE__ . '\Shortcodes\Jobs_Shortcode', 'register_shortcode' ) );
```

Comment:
The `init` hook in particular is always called _after_ `plugins_loaded`, so this `add_action( 'init', ...)` call can probably be moved out of `plugins_loaded` and just be on a line on its own.